test/e2e/storage/csi_mock_volume.go

```diff
@@ -505,8 +505,9 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 					checkPVCSize()
 				} else {
 					ginkgo.By("Checking for conditions on pvc")
```

Comment:
the same logic exists in volume_expand.go#191... does this need to be changed there as well?

Reply:
I thought I changed that in - https://github.com/kubernetes/kubernetes/pull/85297 . line#191 points to something else now.

Reply:
oops, I was on an old branch